.travis.yml

```diff
@@ -16,6 +18,8 @@ env:
 before_script:
   - ./dev/bots/travis_setup.sh
 script:
+  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
```

Comment:
These sure can, I felt it was cleaner to put them here. Will move them to the main script